lib/jgi_mg_assembly/pipeline_steps/step.py

```diff
@@ -19,7 +19,10 @@ def __init__(self, name, version_name, base_command, scratch_dir, output_dir, sh
         self.output_dir = output_dir
         config = ConfigParser()
         config.read("/kb/module/pipeline.cfg")
-        self.version = config.get("versions", version_name)
+        try:
+            self.version = config.get("versions", version_name)
+        except:
```

Comment:
I think I was just lazy there. The ConfigParser will cough up some kind of exception if it can't find that key. Either a NoOptionError or NoSectionError. I can add those there.

Basically, I was thinking that if it can't find the version of whatever the Step is supposed to be called, for any reason, it should just silently call it "version unknown". It shouldn't be a crash-level error.